mkosi

```diff
@@ -2050,7 +2068,7 @@ def secure_boot_sign(args, workspace, run_build_script, for_cache):
                 os.rename(p + ".signed", p)
 
 def xz_output(args, raw):
-    if args.output_format not in (OutputFormat.raw_btrfs, OutputFormat.raw_gpt, OutputFormat.raw_squashfs):
+    if args.output_format not in (RAW_FORMATS):
```

Comment:
there shouldn't be () around this, no?

Reply:
Yes. Am totally not a Python guru. I've fixed these up throughout.